src/dotnet/APIView/APIViewWeb/README.md

```diff
@@ -0,0 +1,80 @@
+# APIView
+
+APIView tool is used by archboard reviewers to review API signatures of all public APIs available in Azure SDK packages. This tool generates public API surface level review which shows all publicly available classes, methods, properties etc. This makes it easier to identify if there are any breaking changes. Currently APIView tool supports following languages:
+
+- C#
+- C
+- C++
+- Java
+- JS/TS
+- Python
+- Go
+- Swift
+
+## Why do we need APIView
+
+APIView tool allows to see only stub version of classes, methods, properties, method signatures available in each Azure SDK. This helps architects to review an Azure SDK and helps to identify any potential change that can impact consumers of an Azure SDK. APIView is also used as an enforcing tool to make sure we release a GA version of Azure SDK package only after it is approved by an architect. APIView is also utlized to identify any change at API level in a pull request.
+
+
+## How to create an API review manually
+
+API review can be created by uploading an artifact to APIView tool. Type of the artifact is different across each language. Some language, for e.g., Swift requires developer to run parser tool locally to generate stub file and upload json stub file instead of any artifact. Following are the detailed instruction on hwo to create review for each language. 

```

Comment:
```suggestion
API review can be created by uploading an artifact to APIView tool. Type of the artifact is different across each language. Some language, for e.g., Swift requires developer to run parser tool locally to generate stub file and upload json stub file instead of any artifact. Following are the detailed instruction on two to create review for each language. 

```